src/schema.js

```diff
@@ -258,14 +278,40 @@ export const accumulateDifferentialErrors = (
                 newObj?.[fieldName],
                 oldError?.fields?.[fieldName],
                 localFields[fieldName],
+                newObj,
             );
+
             if (fieldError) {
                 if (!errors.fields) {
                     errors.fields = {};
                 }
                 errors.fields[fieldName] = fieldError;
             }
+
+            if (dependents[fieldName]) {
```

Comment:
Why check for dependencies? Why not just check if any of the dependencies has changed instead of this apporach?